src/bindings/python/src/pyopenvino/graph/node_input.cpp

```diff
@@ -91,6 +99,16 @@ void regclass_graph_Input(py::module m) {
                 get_rt_info : RTMap
                     A dictionary of user defined data.
              )");
+    input.def("replace_source_output",
+              &ov::Input<ov::Node>::replace_source_output,
+              py::arg("new_source_output"),
+              R"(
+                Replaces the source output of this input.
+
+                Returns
+                ----------
+                None
```

Comment:
```suggestion
                Replaces the source output of this input.
                
                :param new_source_output: New source output.
                :type new_source_output: openvino.runtime.Output
```
Please use new formatting when adding docs.

Reply:
Updated. Thank you!